### PR TITLE
Dockerfile: using upstream and non-privileged ports

### PR DESCRIPTION
**Pull request by bryanlatten**
BREAKING: no longer uses port 80 inside the container, a privileged port

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -101,4 +101,6 @@ RUN sed -i "s/listen = .*/listen = 127.0.0.1:9000/" $CONF_FPMPOOL && \
 COPY ./container/root /
 
 # Override default ini values for both CLI + FPM
-RUN php5enmod overrides
+RUN php5enmod overrides && \
+    # Set nginx to listen on defined port \
+    sed -i "s/listen [0-9]*;/listen ${CONTAINER_PORT};/" $CONF_NGINX_SITE
```
Comment (bryanlatten) [inline Dockerfile:106]: this is also done in the parent, but this container layers its own nginx conf ontop

**Comment by ingluisjimenez** [comment]
👍

**Comment by jimdelois** [comment]
👍

**Comment by bossjones** [comment]
👍

**Comment by mannytoledo** [comment]
👍

**Pull request merged**